### ISSUE TITLE
Issue with launching 1.18 - No LWJGL being loaded

### ISSUE THREAD

**Issue by marewey** [mindstorm38/portablemc#236]
I noticed this issue with a modpack I have with PMC, I then tried removing all libraries, assets, etc. That did not solve the issue.
I tried removing all mods, issue persisted, It simply closes and does not launch
I tried launching with a whole new directory
I tried launching 1.19.2, and has no issues, I tried launching 1.12.2, still works.
Finally realized what the issue was: Unrecognized user type

I also notice in the log "--userType, ," I looked on PMC for an option to specify this, but there is none.

At this point, I believe the issue lies within PMC itself
This did work previously, so it must have been something in an update to PMC

Having same issue with 1.18 fabric, forge, and vanilla

Here is the log (I used ************ to censor personal info):

```
[←[92m  OK  ←[0m] Loaded version forge-1.18.2-40.2.9
[←[92m  OK  ←[0m] Loaded version 1.18.2
[←[92m  OK  ←[0m] Loaded Mojang java 17.0.1.12.1
[←[92m  OK  ←[0m] Checked version jar
[←[92m  OK  ←[0m] Checked 3071 assets version 1.18
[←[92m  OK  ←[0m] Checked 56 class and 8 native libraries
[←[92m  OK  ←[0m] Using logger client-1.12

[21:59:55] [main] [INFO] ModLauncher running: args [--username, ************, --version, forge-1.18.2-40.2.9, --gameDir, ************, --assetsDir, ************\assets, --assetIndex, 1.18, --uuid, ************, --accessToken, ❄❄❄❄❄❄❄❄, --clientId, , --xuid, , --userType, , --versionType, release, --launchTarget, forgeclient, --fml.forgeVersion, 40.2.9, --fml.mcVersion, 1.18.2, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20220404.173914]
[21:59:55] [main] [INFO] ModLauncher 9.1.3+9.1.3+main.9b69c82a starting: java version 17.0.1 by Microsoft
[21:59:55] [main] [INFO] SpongePowered MIXIN Subsystem Version=0.8.5 Source=union:/************/libraries/org/spongepowered/mixin/0.8.5/mixin-0.8.5.jar%2314!/ Service=ModLauncher Env=CLIENT
←[33m[21:59:55] [main] [WARN] Mod file ************\libraries\net\minecraftforge\fmlcore\1.18.2-40.2.9\fmlcore-1.18.2-40.2.9.jar is missing mods.toml file
←[0m←[33m[21:59:55] [main] [WARN] Mod file ************\libraries\net\minecraftforge\javafmllanguage\1.18.2-40.2.9\javafmllanguage-1.18.2-40.2.9.jar is missing mods.toml file
←[0m←[33m[21:59:55] [main] [WARN] Mod file ************\libraries\net\minecraftforge\lowcodelanguage\1.18.2-40.2.9\lowcodelanguage-1.18.2-40.2.9.jar is missing mods.toml file
←[0m←[33m[21:59:55] [main] [WARN] Mod file ************\libraries\net\minecraftforge\mclanguage\1.18.2-40.2.9\mclanguage-1.18.2-40.2.9.jar is missing mods.toml file
←[0m[21:59:55] [main] [INFO] Found mod file fmlcore-1.18.2-40.2.9.jar of type LIBRARY with provider net.minecraftforge.fml.loading.moddiscovery.MinecraftLocator@5a6d5a8f
[21:59:55] [main] [INFO] Found mod file javafmllanguage-1.18.2-40.2.9.jar of type LANGPROVIDER with provider net.minecraftforge.fml.loading.moddiscovery.MinecraftLocator@5a6d5a8f
[21:59:55] [main] [INFO] Found mod file lowcodelanguage-1.18.2-40.2.9.jar of type LANGPROVIDER with provider net.minecraftforge.fml.loading.moddiscovery.MinecraftLocator@5a6d5a8f
[21:59:55] [main] [INFO] Found mod file mclanguage-1.18.2-40.2.9.jar of type LANGPROVIDER with provider net.minecraftforge.fml.loading.moddiscovery.MinecraftLocator@5a6d5a8f
[21:59:55] [main] [INFO] Found mod file client-1.18.2-20220404.173914-srg.jar of type MOD with provider net.minecraftforge.fml.loading.moddiscovery.MinecraftLocator@5a6d5a8f
[21:59:55] [main] [INFO] Found mod file forge-1.18.2-40.2.9-universal.jar of type MOD with provider net.minecraftforge.fml.loading.moddiscovery.MinecraftLocator@5a6d5a8f
[21:59:55] [main] [INFO] No dependencies to load found. Skipping!
[21:59:56] [main] [INFO] Launching target 'forgeclient' with arguments [--version, forge-1.18.2-40.2.9, --gameDir, ************, --assetsDir, ************\assets, --uuid, d51eb0e3df0652a0a61fef6daa858f7a, --username, System, --assetIndex, 1.18, --accessToken, ❄❄❄❄❄❄❄❄, --clientId, , --xuid, , --userType, , --versionType, release]
←[33m[22:00:01] [main] [WARN] Unrecognized user type:
←[0m
```

Here is the log for 1.18.2 vanilla:

```
[←[92m  OK  ←[0m] Loaded version 1.18.2
[←[92m  OK  ←[0m] Loaded Mojang java 17.0.1.12.1
[←[92m  OK  ←[0m] Checked version jar
[←[92m  OK  ←[0m] Checked 3071 assets version 1.18
[←[92m  OK  ←[0m] Checked 29 class and 8 native libraries
[←[92m  OK  ←[0m] Using logger client-1.12

←[33m[22:25:42] [main] [WARN] Unrecognized user type:
←[0m[22:25:43] [Render thread] [INFO] [STDERR]: java.lang.NoClassDefFoundError: Could not initialize class com.mojang.blaze3d.systems.RenderSystem
[22:25:43] [Render thread] [INFO] [STDERR]:     at ac.a(SourceFile:65)
[22:25:43] [Render thread] [INFO] [STDERR]:     at dyr.a(SourceFile:2394)
[22:25:43] [Render thread] [INFO] [STDERR]:     at dyr.a(SourceFile:2389)
[22:25:43] [Render thread] [INFO] [STDERR]:     at net.minecraft.client.main.Main.main(SourceFile:206)
```

**Comment by mindstorm38** [comment]
I'm quite sure this is not related to the unrecognized user type, I've already seen that in many versions and is only related to the fact that you are no launcher with an authentication session and MC don't recognize it, which is logic. 

I'll try to reproduce this bug, because I don't remember having issues with 1.18.2, and this class not found is weird, because blaze3d is either embedded directly in the version's client Jar, or is a library that is checked to be installed before any launch, so it should exists!

**Comment by mindstorm38** [comment]
Ok I've the same issue, I'm working on this!

**Comment by mindstorm38** [comment]
So this is definitely an issue with recent versions, none of the provided LWJGL version was loaded, but I wonder if the fix can break something that we fixed in the past...

**Comment by mindstorm38** [comment]
@marewey This is fixed in latest release 4.4.1!

**Comment by marewey** [comment]
Thank you! Glad to have it fixed!
